### ISSUE TITLE
canary failing on 1.5.x

### ISSUE THREAD

**Issue by odedniv** [angular-translate/angular-translate#1552]
### Subject of the issue
canary branch is failing when running `npm run test-scopes`.

### Your environment
Not applicable (test suite).

### Steps to reproduce
```
npm run test-scopes
```

### Expected behaviour
Should not fail.

### Actual behaviour
All angular versions are fine except 1.5.x, which gives out:
```
Currently selected scope:  angular_1.5.x
04 08 2016 13:29:43.107:INFO [karma]: Karma v0.13.22 server started at http://localhost:9876/
04 08 2016 13:29:43.113:INFO [launcher]: Starting browser Chrome
04 08 2016 13:29:43.772:INFO [Chrome 52.0.2743 (Linux 0.0.0)]: Connected on socket /#lGCwvim1acjMbydnAAAA with id 68281284
LOG: 'setting timeout to', 9
LOG: 'setting timeout to', 46
LOG: 'setting timeout to', 4
Chrome 52.0.2743 (Linux 0.0.0) pascalprecht.translate $translate#use() with async loading and two unordered requests in parallel (1st win) should be requested the first language FAILED
        Error: No deferred tasks to be flushed
            at Function.self.defer.flush (/home/oded/devel/angular-translate/test_scopes/angular_1.5.x/bower_components/angular-mocks/angular-mocks.js:127:15)
            at Function.$delegate.flush (/home/oded/devel/angular-translate/test_scopes/angular_1.5.x/bower_components/angular-mocks/angular-mocks.js:2081:20)
            at Object.<anonymous> (/home/oded/devel/angular-translate/test/unit/service/translate.spec.js:716:16)
            at Object.invoke (/home/oded/devel/angular-translate/test_scopes/angular_1.5.x/bower_components/angular/angular.js:4718:19)
            at Object.workFn (/home/oded/devel/angular-translate/test_scopes/angular_1.5.x/bower_components/angular-mocks/angular-mocks.js:3085:20)
        Error: Declaration Location
            at window.inject.angular.mock.inject (/home/oded/devel/angular-translate/test_scopes/angular_1.5.x/bower_components/angular-mocks/angular-mocks.js:3047:25)
            at Suite.<anonymous> (/home/oded/devel/angular-translate/test/unit/service/translate.spec.js:702:16)
            at Suite.<anonymous> (/home/oded/devel/angular-translate/test/unit/service/translate.spec.js:654:3)
            at /home/oded/devel/angular-translate/test/unit/service/translate.spec.js:5:1
Chrome 52.0.2743 (Linux 0.0.0) pascalprecht.translate $translate#use() with async loading and two unordered requests in parallel (1st win) should be requested the second language FAILED
        Error: No deferred tasks to be flushed
            at Function.self.defer.flush (/home/oded/devel/angular-translate/test_scopes/angular_1.5.x/bower_components/angular-mocks/angular-mocks.js:127:15)
            at Function.$delegate.flush (/home/oded/devel/angular-translate/test_scopes/angular_1.5.x/bower_components/angular-mocks/angular-mocks.js:2081:20)
            at Object.<anonymous> (/home/oded/devel/angular-translate/test/unit/service/translate.spec.js:716:16)
            at Object.invoke (/home/oded/devel/angular-translate/test_scopes/angular_1.5.x/bower_components/angular/angular.js:4718:19)
            at Object.workFn (/home/oded/devel/angular-translate/test_scopes/angular_1.5.x/bower_components/angular-mocks/angular-mocks.js:3085:20)
Chrome 52.0.2743 (Linux 0.0.0) pascalprecht.translate $translate#use() with async loading and two unordered requests in parallel (1st win) should set the language to be the most recently requested one, not the most recently responded one FAILED
        Error: No deferred tasks to be flushed
            at Function.self.defer.flush (/home/oded/devel/angular-translate/test_scopes/angular_1.5.x/bower_components/angular-mocks/angular-mocks.js:127:15)
            at Function.$delegate.flush (/home/oded/devel/angular-translate/test_scopes/angular_1.5.x/bower_components/angular-mocks/angular-mocks.js:2081:20)
            at Object.<anonymous> (/home/oded/devel/angular-translate/test/unit/service/translate.spec.js:716:16)
            at Object.invoke (/home/oded/devel/angular-translate/test_scopes/angular_1.5.x/bower_components/angular/angular.js:4718:19)
            at Object.workFn (/home/oded/devel/angular-translate/test_scopes/angular_1.5.x/bower_components/angular-mocks/angular-mocks.js:3085:20)
Chrome 52.0.2743 (Linux 0.0.0): Executed 446 of 446 (3 FAILED) (1.9 secs / 1.804 secs)
```

**Comment by tspaeth** [comment]
If you take a look into the Travis Log it seems to fail from time to time the last builds. Locally, it runs here. So it looks more like a problem on the build server. Could be something locally also, but the errors on the build server are not really consistent.

**Comment by knalli** [comment]
@tspaeth Ensure you have the latest 1.5.x version in the test scope: `npm run clean-test-scopes`

**Comment by tspaeth** [comment]
@knalli Yup. No issues locally.

**Comment by odedniv** [comment]
I am getting this error locally as well. I got it before creating the pull request, but I thought it was a local problem and now I see Travis is getting this as well.

It's a fresh clone of the project, and `node_modules/angular/package.json` state `1.5.8`. Running `npm run clean-test-scopes` didn't help.

Bottom line, it's not just Travis.

**Comment by tspaeth** [comment]
Hm. Ok. I'll check out to a clean folder and try again with that one.

**Comment by tspaeth** [comment]
after using a clean workspace and running
```
grunt run-script test-scope
```

I was able to reproduce the issue.

**Comment by JamesHenry** [comment]
An important update to this is that Chrome 52 is no longer Canary - it is **stable**.

Moving between 51 and 52, my production app has now changed in its handling of `Missing Translation` errors when fallbacks are available. It used to silently fallback to the configured fallbackLanguage but now throws an error in every case which is hammering my logs.

To confirm: No code changes were made, the behaviour changes purely by changing browsers.

**Comment by knalli** [comment]
Ehm.. wrong issue? :)

**Comment by JamesHenry** [comment]
New issue created: https://github.com/angular-translate/angular-translate/issues/1554

**Comment by knalli** [comment]
Breakdown: Issue is introduced due angular-mocks 1.5.7 -> 1.5.8

**Comment by knalli** [comment]
I have applied an explicitly delay covering everything. Strange, but I have also not much opportunity debugging angular-mocks.
